website/docs/r/container_app.html.markdown

```diff
@@ -81,9 +81,15 @@ The following arguments are supported:
 
 A `secret` block supports the following:
 
+* `identity` - (Optional) The identity to use for accessing Key Vault reference (Required if `key_vault_url` is used).
+
+* `key_vault_url` - (Optional) The URL to a Key Vault secret.
```

Comment:
Can we remove the above *Required if `key_vault_url` is used* and put the following below:

```
*NOTE:* -> `identity` must be used together with `key_vault_url`
```

Reply:
updated